src/1_Preproc.r

```diff
@@ -34,7 +34,7 @@ suppressWarnings(library(MASS))
 #' @return TRUE if the design is correct FALSE otherwise
 check_10x_input <- function(samples){
   
-  cell_ranger_v2 <- c("genes.tsv", "barcodes.tsv", "matrix.mtx")
+  cell_ranger_v2 <- c("genes.tsv.gz", "barcodes.tsv.gz", "matrix.mtx.gz")
```

Comment:
```suggestion
  # Cell ranger v2 does in fact produce non gunzipped outputs, but since during
  # data-upload gzip compresses all non-compressed file, so we expect files with *.gz ending here
  cell_ranger_v2 <- c("genes.tsv.gz", "barcodes.tsv.gz", "matrix.mtx.gz")
```
Just a suggestions to point out this hacky workaround